sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample_LROPolling.md

```diff
@@ -0,0 +1,149 @@
+# Polling the Long Running Operations
```

Comment:
Hmmm this looks like it should belong in the .NET repo itself so multiple libraries can consume it and not just in TA.
Could you file an issue in the repo mentioning this?
When it gets done, you could point to that instead of this :) but while that happens, thanks for writing this

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/17475